src/v2/cookbook/serverless-blog.md

```diff
@@ -88,7 +89,7 @@ export default new Router({
 
 Then create `components/BlogHome.vue` which will be your blog homepage that lists your most recent posts.
 
-```javascript
+```html
```

Comment:
The code snippet seems should be used with Vue syntax, not JavaScript. And as the `vue` syntax is not supported in Hexo, so I changed it into `html` as a downgrade syntax.

Reply:
Huh, interesting